<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_gamePaused</class>
 <widget class="QDialog" name="dialog_gamePaused">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_gamePaused">
     <property name="text">
      <string>Game Paused</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_unPause">
        <property name="text">
         <string>Resume</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CancelGame">
        <property name="text">
         <string>Cancel Game</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
